### PR TITLE
Add .NET Standard support

### PR DESCRIPTION
**Pull request by jvitkauskas**
Because that's what we (.NET devs) do all day long.

### CODE REVIEW

**Comment by gregoryyoung** [comment]
:+1:

File: Unit/Unit.csproj [hunk 1]
```diff
@@ -1,53 +1,12 @@
...
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{72CCF8C1-2A26-4DBE-8D78-75E0BDF5A091}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Unit</RootNamespace>
-    <AssemblyName>Unit</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
+    <TargetFrameworks>net45;netstandard1.0</TargetFrameworks>
```
Comment (dsyme) [inline Unit/Unit.csproj:4]: It would be better if this list of TargetFrameworks was empty.  Nothing should target nothing.

Comment (dsyme) [reply Unit/Unit.csproj:4]: Or try

    <TargetFrameworks>nothing</TargetFrameworks>

Comment (bartelink) [reply Unit/Unit.csproj:4]: Indeed; Nothing will come from nothing, you know what they say?

**Pull request merged**